src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,59 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private String by;
```

Comment:
endDateString should be a more appropriate name here
```suggestion
    private String endDateString;
```